ocpp/v201/enums.py

```diff
@@ -1350,3 +1351,1347 @@ class SecurityEventType(str, Enum):
+    tripped = "Tripped"
+
+
+class RCDRecloserVariableName(str, Enum):
+    """
+    Variable names where the component type is RCDRecloser
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    active_set = "Active(Set)"
+    complete = "Complete"
+    enabled = "Enabled"
+    problem = "Problem"
+    tries = "Tries"
+    tries_max_limit = "Tries(MaxLimit)"
+    tries_set_limit = "Tries(SetLimit)"
+
+
+class RealTimeClockVariableName(str, Enum):
+    """
+    Variable names where the component type is RealTimeClock
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    dc_voltage = "DCVoltage"
+    fallback = "Fallback"
+    fallback_max_limit = "Fallback(MaxLimit)"
+    problem = "Problem"
+
+
+class ShockSensorVariableName(str, Enum):
+    """
+    Variable names where the component type is ShockSensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    force = "Force"
+
+
+class SpacesCountSignageVariableName(str, Enum):
+    """
+    Variable names where the component type is SpacesCountSignage
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    count = "Count"
+    enabled = "Enabled"
+
+
+class SwitchVariableName(str, Enum):
+    """
+    Variable names where the component type is Switch
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    state = "State"
+
+
+class TemperatureSensorVariableName(str, Enum):
+    """
+    Variable names where the component type is TemperatureSensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    problem = "Problem"
+    temperature = "Temperature"
+
+
+class TiltSensorVariableName(str, Enum):
+    """
+    Variable names where the component type is TiltSensor
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    angle = "Angle"
+
+
+class TokenReaderVariableName(str, Enum):
+    """
+    Variable names where the component type is TokenReader
+    See PhysicalComponentName for referenced physical component
+    """
+
+    enabled = "Enabled"
+    enabled_set = "EnabledSet"
+    operated = "Operated"
+    problem = "Problem"
+    token = "Token"
+    token_type = "TokenType"
+
+
+class UIInputVariableName(str, Enum):
+    """
+    Variable names where the component type is UIInput
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    enabled = "Enabled"
+    operated = "Operated"
+
+
+class UpstreamProtectionTriggerVariableName(str, Enum):
+    """
+    Variable names where the component type is UpstreamProtectionTrigger
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active_set = "ActiveSet"
```

Comment:
In In OCPP-2.0.1_part2_appendices_v13.pdf, 3.2.54. UpstreamProtectionTrigger that's `Active(Set)`.

Reply:
Corrected